composer.json

```diff
@@ -29,7 +29,7 @@
         "mimmi20/file-loader": "~2.0",
         "symfony/filesystem": "*",
         "symfony/finder": "~2.6",
-        "symfony/console": "~2.6",
+        "symfony/console": "^3.0",
```

Comment:
Could we use `^2.6|^3.0` to ensure nothing conflicts for now please? I can see us removing 2.x at some point in the future though.